src/main/java/games/strategy/engine/lobby/server/login/AccessLog.java

```diff
@@ -1,12 +1,11 @@
 package games.strategy.engine.lobby.server.login;
 
-import java.net.InetAddress;
+import java.time.Instant;
+
+import games.strategy.engine.lobby.server.User;
 
-/**
- * A lobby access log.
- */
 interface AccessLog {
-  void logFailedLogin(LoginType loginType, String username, InetAddress address, String errorMessage);
+  void logFailedAccess(Instant instant, User user, AccessMethod accessMethod, String errorMessage);
 
-  void logSuccessfulLogin(LoginType loginType, String username, InetAddress address);
```

Comment:
Yeah, I kind of viewed _access_ as being implied by _login_ or _authentication_, because one typically isn't granted access until they are authenticated.  However, I agree the term does not fit in with our domain model, so I will rename these methods to remove the `Access` suffix.  However, I think I'm going to use `Authentication` instead of `Login` just to be consistent with the other terminology we've been moving to inside `LobbyLoginValidator` over the past several months.